autogen/sr.rs

```diff
@@ -374,6 +374,9 @@ pub fn gen_sr_code_from_instruction_grammar(
                         }),
                     });
                 }
+            }
+            _ if inst_name == "Phi" => {
+
```

Comment:
I'm assuming this is a purposeful "ignore Phi instructions", if it is, a comment briefly explaining that (and maybe where they do get handled) would be a good idea.

Reply:
will do